### PR TITLE
3.x: Fix wrong description for bean validation annotations (#8667)

### PR DESCRIPTION
**Pull request by Captain1653**
### Description
Resolves https://github.com/helidon-io/helidon/issues/8667

I followed [Jakarta Bean Validation Specification](https://jakarta.ee/specifications/bean-validation/3.0/jakarta-bean-validation-spec-3.0.html)

### CODE REVIEW

File: docs/mp/beanvalidation.adoc [hunk 1]
```diff
@@ -191,6 +193,7 @@ Supported types are:
 |`@Size`
 a|The annotated element size must be between the specified boundaries (included).
 Supported types are:
+
```
Comment (Captain1653) [inline docs/mp/beanvalidation.adoc:196]: Without this empty line content looks a little wrong.
Example:
Screenshot 2024-04-18 at 18 49 37

With this empty line:
Screenshot 2024-04-18 at 18 50 46

**Comment by Captain1653** [comment]
@spericas Can you review backport for 3.x? Please :)

**Review by romain-grecourt — APPROVED** [review]

**Comment by Captain1653** [comment]
Some checks are failed, but I didn't touch them.
Screenshot 2024-04-26 at 07 16 57

**Comment by barchetta** [comment]
@Captain1653 you'll need to rebase this PR with the latest from the helidon-3.x branch.

The example build failure is caused by the combination of examples having moved out of this repository and the fact that we did a 3.x release and bumped the 3.x version while this PR has been open.  So you need to pick up the pom version change to 3.2.9-SNAPHOST.

**Comment by Captain1653** [comment]
@barchetta Thank you for the explanation.

Done.

**Review by romain-grecourt — APPROVED** [review]

**Pull request merged**